softlearning/algorithms/sql.py

```diff
@@ -216,10 +217,10 @@ def _create_td_update(self):
     def _create_svgd_update(self):
         """Create a minimization operation for policy update (SVGD)."""
 
-        actions = self.policy.actions_for(
-            observations=self._observations_ph,
-            n_action_samples=self._kernel_n_particles,
-            reuse=True)
+        action_tensors = [tf.expand_dims(self.policy.actions(self._observations_ph), -2)
+                          for _ in range(self._kernel_n_particles)]
+        actions = tf.concat(action_tensors, 1)
```

Comment:
@hartikainen these fixes are very hack-y. However, I haven't come up with a way to fix it.

Reply:
This doesn't seem to work :(

Reply:
When you say it doesn't work, does it crash or does it not learn?

Reply:
Seems like this works fine. I pushed this over to your branch.